src/components/settings/Settings.h

```diff
@@ -9,6 +9,7 @@ namespace Pinetime {
     class Settings {
     public:
       enum class ClockType : uint8_t { H24, H12 };
+      enum class DateType : uint8_t { DMY, MDY };
```

Comment:
It might be interesting to try and combine these, so the first bit would determine the clock type, and the second the date month order. Then settingsVersion wouldn't have to be updated.

Reply:
I considered this idea. The only reason I didn't do it is because the watchfaces would need to be refactored for that change.

I think it's a great change, but it means we need to make sure to update all the watchfaces in this PR. I'm more than happy to do that, if this is something we wanted to move forward with.

Reply:
They don't necessarily need to be refactored, if the public enum classes and function behaviors are retained. The fact that they're combined in a single variable doesn't need to be visible outside of the class.

Reply:
Ah, that's true. I'll combine them then.